src/parser/cvc/Cvc.g

```diff
@@ -598,6 +598,8 @@ using namespace CVC4::parser;
 #define SOLVER PARSER_STATE->getSolver()
 #undef MK_TERM
 #define MK_TERM SOLVER->mkTerm
+#undef MK_SINGLETON
+#define MK_SINGLETON SOLVER->mkSingleton
```

Comment:
Since this is only used twice, I'd prefer if it were inlined.

Reply:
Done